docs/updating-ccid.md

```diff
@@ -0,0 +1,77 @@
+# Updating the CCID free software driver
+
+Note: This page is mainly intended for the Chrome OS Smart Card Connector
+project maintainers.
+
+## Background
+
+The Chrome OS Smart Card Connector app is shipped with a specific version of the
+CCID free software driver. This is a driver for smart card reader (not to be
+confused with drivers for smart cards, also sometimes called middleware).
+
+Reasons for pinning to a specific version of the driver:
```

Comment:
nit: Here you imply that we are using a pinned version of CCID, but it may be better to state it outright in this line. (I think when you state above that SCC is shipped with a specific version, it is not yet clear that the version in the repo is also pinned)